app/frontend/src/styles/base/_utilities.scss

```diff
@@ -183,10 +130,6 @@ input.button-as-link {
   }
 }
 
-.formatable-textarea {
-  min-height: 150px;
-}
-
 ul li + br,
```

Comment:
i think this was related to

Reply:
related to?

Reply:
related to

Reply:
lol, I obviously got halfway through a thought!